Structure_Engine/Query/StructuralMaterialFragment.cs

```diff
@@ -36,7 +36,9 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Gets a structural material fragment from a material class")]
+        [Description("Gets a structural material fragment, containing all relevant structural material data, from a physical Material class.")]
```

Comment:
Material

Reply:
Not sure we should capitalise the material here, except where it is already capitalised. The rest refers to the structures oM where we do not have any class named Material